tracker/tracker-vuex/src/index.ts

```diff
@@ -22,25 +46,26 @@ export default function(opts: Partial<Options> = {}) {
     }
     const encoder = new Encoder(sha1, 50);
     return store => {
-      store.subscribe((mutation, state) => {
-        if (options.filter(mutation, state)) {
+      // Vuex
+      if (store.subscribe) {
+        store.subscribe((mutation, state) => {
+          processMutationAndState(app, options, encoder, mutation, state);
+        });
+      }
+      // Pinia
+      if (store.$onAction) {
+        store.$onAction(({ name, store, args }) => {
           try {
-            const { type } = mutation;
-            if (typeof type === 'string' && type) {
-              app.send(Messages.StateAction(type));
-            }
-            const _mutation = encoder.encode(
-              options.mutationTransformer(mutation)
-            );
-            const _state = encoder.encode(options.transformer(state));
-            const _table = encoder.commit();
-            for (let key in _table) app.send(Messages.OTable(key, _table[key]));
-            app.send(Messages.Vuex(_mutation, _state));
-          } catch {
-            encoder.clear();
-          }
-        }
-      });
+            let state = {};
```

Comment:
If we use it as a container for many stores, it should be defined before the `$onAction`  subscription, otherwise I suspect that  diffs won't be displayed correctly.

Reply:
Also It will reduce amount of the information  sent due to the way Encoder works

Reply:
That is a good idea  since right now it can only show a state of a one store. There can be many stores even in pure vuex, as far as I know.  So we can extend this to the original `subscribe` having  a `storeName: string` as a second  argument (line 24/48).